tfe/provider_test.go

```diff
@@ -103,6 +103,7 @@ var GITHUB_WORKSPACE_IDENTIFIER = os.Getenv("GITHUB_WORKSPACE_IDENTIFIER")
 var GITHUB_WORKSPACE_BRANCH = os.Getenv("GITHUB_WORKSPACE_BRANCH")
 var GITHUB_POLICY_SET_IDENTIFIER = os.Getenv("GITHUB_POLICY_SET_IDENTIFIER")
 var GITHUB_POLICY_SET_BRANCH = os.Getenv("GITHUB_POLICY_SET_BRANCH")
+var GITHUB_POLICY_SET_ALT_BRANCH = os.Getenv("GITHUB_POLICY_SET_ALT_BRANCH")
```

Comment:
nit: would it be possible to avoid introducing another env var here? there are getting unwieldy. i'd love to find an alternative, if there is one, but it is pretty low priority. 

the tfe_workspace tests also have a vcs repo and test going from master to GITHUB_WORKSPACE_BRANCH. is that something that would be possible to do here as well?

Reply:
@lafentres I've made a few changes to the vcs related tests so as to move away from our reliance on env vars. Take a look and let me know what you think.